src/Controller/API/Competencies.php

```diff
@@ -34,7 +34,7 @@ public function getOne(
     }
 
     /**
-     * @Route("/api/{version<v1|v3>}/competencies", methods={"GET"})
+     * @Route("/api/{version<v3>}/competencies", methods={"GET"})
      */
     public function getAll(
```

Comment:
Based on [559eb323aa79bf66f12af092c27a24d72f931350] this (and several other methods in other controllers) were added to restrict write access and can now be removed.

Reply:
:heavy_check_mark: